lib/service_locator.dart

```diff
@@ -1,3 +1,5 @@
+// Simplifies usage of service locator greatly.
+//ignore_for_file: no-equal-arguments
```

Comment:
How does ignoring this rule help simplify the usage of sl? I don't see any linter errors by removing this line